kernel-spark/src/test/java/io/delta/kernel/spark/read/SparkGoldenTableTest.java

```diff
@@ -60,10 +60,10 @@ public void setUp(@TempDir File tempDir) {
         new SparkConf()
             .set("spark.sql.catalog.dsv2", "io.delta.kernel.spark.catalog.TestCatalog")
             .set("spark.sql.catalog.dsv2.base_path", tempDir.getAbsolutePath())
-            .set("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension")
+            .set("spark.sql.extensions", "io.delta.sql.LegacyDeltaSparkSessionExtension")
```

Comment:
How about renaming all the `Legacy` as V1 instead?

Reply:
I prefer "Legacy" as explicitly calling "Legacy" could make people avoid using it(except for test).

Reply:
on a different note, I dont have a strong opinion on Legacy vs v1 as those are internal names... but V1/V2 is more consistent with the sparkV1 and sparkV2 modules. and we can we be consistent in other places by using v1 and v2.